scripts/development/build-contenta_jsonapi.sh

```diff
@@ -44,7 +45,7 @@ echo "-----------------------------------------------"
 echo " Downloading Contenta CMS using composer "
 echo "-----------------------------------------------"
 echo -e "${FG_C}${BG_C} EXECUTING ${NO_C} $COMPOSER create-project contentacms/contenta-jsonapi-project ${DEST_DIR} --stability dev --no-interaction\n\n"
-$COMPOSER create-project contentacms/contenta-jsonapi-project ${DEST_DIR} --stability dev --no-interaction --no-install
+$PHP -d memory_limit=-1 $COMPOSER create-project contentacms/contenta-jsonapi-project ${DEST_DIR} --stability dev --no-interaction --no-install
```

Comment:
Is this necessary after altering the `.ini` file? Are you thinking about the user's local machine and not just Travis?